Case Sensitivity issues with "./localStorage-ttls"

Getting the following error on `npm start`

```
ERROR in ./~/ampersand-local-cache-mixin/ampersand-local-cache-mixin.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./localStorage-ttls in /var/www/node/ghublabler/node_modules/ampersand-local-cache-mixin
 @ ./~/ampersand-local-cache-mixin/ampersand-local-cache-mixin.js 6:14-44
```

This is due to Line 6 of ampersand-local-cache-mixin.js:

```
var storage = require('./localStorage-ttls')
```

The file is installed as "localstorage-ttls.js".  This causes a file not found error.

--- a/ampersand-local-cache-mixin.js
+++ b/ampersand-local-cache-mixin.js
@@ -3,7 +3,7 @@ var isString = require('lodash.isstring')
 var result = require('lodash.result')
 var merge = require('lodash.merge')
 var pick = require('lodash.pick')
-var storage = require('./localStorage-ttls')
+var storage = require('./localstorage-ttls')
 
 module.exports = {
   initStorage: function (opts) {

